File: 0380-insert-delete-getrandom-o1/0380-insert-delete-getrandom-o1.scala
class RandomizedSet() {
    val ls= scala.collection.mutable.Set[Int]()
    def insert(`val`: Int): Boolean = {
        ls.add(`val`)
    }

    def remove(`val`: Int): Boolean = {
        ls.remove(`val`)
    }

    def getRandom(): Int = {
        val n = util.Random.nextInt(ls.size)
        ls.iterator.drop(n).next()
    }

}

/**
 * Your RandomizedSet object will be instantiated and called as such:
 * var obj = new RandomizedSet()
 * var param_1 = obj.insert(`val`)
 * var param_2 = obj.remove(`val`)
 * var param_3 = obj.getRandom()
 */

File: 0380-insert-delete-getrandom-o1/0380-insert-delete-getrandom-o1.java
class RandomizedSet {
    List<Integer> ls;
    public RandomizedSet() {
        ls= new ArrayList<>();
    }
    
    public boolean insert(int val) {
        if(ls.contains(val)){
            return false;
        }
        ls.add(val);
        return true;
    }
    
    public boolean remove(int val) {
        if(ls.contains(val)){
            ls.remove((Object)val);
            return true;
        }
        return false;
    }
    
    public int getRandom() {
        int index = new Random().nextInt(0, ls.size());
        return ls.get(index);
    }
}

/**
 * Your RandomizedSet object will be instantiated and called as such:
 * RandomizedSet obj = new RandomizedSet();
 * boolean param_1 = obj.insert(val);
 * boolean param_2 = obj.remove(val);
 * int param_3 = obj.getRandom();
 */